lectures/markov_chains_II.md

```diff
@@ -196,16 +196,16 @@ This gives us another way to interpret the stationary distribution (provided irr
 
 Importantly, the result is valid for any choice of $\psi_0$.
 
-The theorem is related to {doc}`the Law of Large Numbers <lln_clt>`.
+The theorem is related to {doc}`the law of large numbers <lln_clt>`.
```

Comment:
@jstac re: style guide. Do we want to always use `{doc}` without replacing the title text (i.e. leaving the title text from the lecture it references) or have a more nuanced approach using replacement text in some cases where it makes sense. Just thinking of a "rule" we can have in the style guide.

Reply:
Good question @mmcky .  I think we should default to the title text but replace it when clarity is enhanced.  In this case it's enhanced.

Reply:
thanks @jstac I will add a section to our style guide.